Source/src/modules/ModuleProgram.cpp

```diff
@@ -281,6 +313,17 @@ void Hachiko::ModuleProgram::UpdateCamera(const ComponentCamera* camera) const
     UpdateUBO(UBOPoints::CAMERA, sizeof(CameraData), &camera_data);
 }
 
+void Hachiko::ModuleProgram::UpdateCamera(const Frustum& frustum) const
+{
+    CameraData camera_data;
+    camera_data.view = float4x4(frustum.ViewMatrix());
+    camera_data.proj = frustum.ProjectionMatrix();
+    // TODO: Understand why camera_data.view.TranslatePart() does not give the position
+    camera_data.pos = camera_data.view.RotatePart().Transposed().Transform(-camera_data.view.TranslatePart());
```

Comment:
We can directly get the correct position from the component camera probably.

Reply:
In that case we don't have a component camera, because we use a specific frustrum created previously